src/main/resources/db/migration/V4__Create_products_table.sql

```diff
@@ -1,12 +1,12 @@
 CREATE TABLE IF NOT EXISTS products (
-                          id SERIAL PRIMARY KEY,
-                          name VARCHAR(255),
-                          thumbnail_image VARCHAR(255),
-                          image VARCHAR(255),
-                          available BOOLEAN DEFAULT FALSE,
-                          description VARCHAR(255),
-                          created_at TIMESTAMP NOT NULL DEFAULT now(),
-                          updated_at TIMESTAMP(6),
-                          seller_id BIGINT,
-                          FOREIGN KEY (seller_id) REFERENCES sellers(id)
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(255),
+    thumbnail_image VARCHAR(255),
+    image VARCHAR(255),
+    available BOOLEAN DEFAULT FALSE,
+    description VARCHAR(255),
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP(6),
+    category_id BIGINT REFERENCES categories,
+    seller_id BIGINT REFERENCES sellers
```

Comment:
there is gonna be an issue with these two.
for esxample: `categories` is a table, but what we want is: 
the field `category_id` in `products` table to refer to the `id` field in `categories` table.

as bonus, a couple useful links for you :)
- https://stackoverflow.com/a/12315244
- https://stackoverflow.com/a/67421485

Reply:
But these corrections were in accordance with the changes in PSQL 16+ that you recommended to me in the previous comment.